projects/httpcomponents-client/HeaderAddFieldFuzzer.java

```diff
@@ -0,0 +1,18 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+
+import org.apache.hc.client5.http.entity.mime.Header;
+import org.apache.hc.client5.http.entity.mime.MimeField;
+
+public class HeaderAddFieldFuzzer {
+    private static final int MAX_NAME_LENGTH = 300;
+    private static final int MAX_VALUE_LENGTH = 700;
+
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+        final Header header = new Header();
+
+        while (data.remainingBytes() > 0) {
+            header.addField(new MimeField(
```

Comment:
I'm not so convinced of this fuzz target as not much logic is going on in the `addField` method (https://github.com/apache/httpcomponents-client/blob/master/httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/Header.java#L53) 

What were you trying to achieve here?